app/models/provider_interface/provider_applications_filter.rb

```diff
@@ -101,6 +104,28 @@ def status_filter
       }
     end
 
+    def course_type_filter
+      return unless FeatureFlag.active?(:teacher_degree_apprenticeship)
+
+      {
+        type: :checkboxes,
+        heading: I18n.t('provider_interface.filters.course_type.heading'),
+        name: 'course_type',
+        options: [
+          {
+            value: POSTGRADUATE_COURSES_PARAM_NAME,
+            label: I18n.t('provider_interface.filters.course_type.option_one'),
```

Comment:
```suggestion
            label: I18n.t('provider_interface.filters.course_type.postgraduate'),
```
`option_1` seems very vague, same thought for the `option_2` value

Reply:
Done